src/ext/java/org/opentripplanner/ext/transmodelapi/model/stop/MonoOrMultiModalStation.java

```diff
@@ -79,7 +80,7 @@ public String getCode() {
     return code;
   }
 
-  public String getDescription() {
+  public I18NString getDescription() {
```

Comment:
The data fetcher in `StopPlaceType` needs to be updated for this.

Reply:
Done